### PR TITLE
fix(OpenApi): resolve errors and generate openapi file

### PR DESCRIPTION
**Pull request by blizzz**
composer run openapi was not running through

### CODE REVIEW

File: .github/workflows/openapi.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          coverage: none
+          ini-file: development
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Check Typescript OpenApi types
+        id: check_typescript_openapi
+        uses: andstor/file-existence-action@076e0072799f4942c8bc574a82233e1e4d13e9d6 # v3.0.0
+        with:
+          files: "src/types/openapi/openapi*.ts"
```
Comment (juliushaertl) [inline .github/workflows/openapi.yml:36]: We don't have that (yet), but let me quickly check how talk does it, maybe we can adapt it as well for #799

Comment (juliushaertl) [reply .github/workflows/openapi.yml:36]: Pushed a commit

**Review by juliushaertl — APPROVED** [review]

File: .github/workflows/openapi.yml (hunk 1 above)
Comment (blizzz) [reply .github/workflows/openapi.yml:36]: Aye, it was optional for the check however.

**Comment by blizzz** [comment]
[Green smoke](https://github.com/nextcloud/tables/actions/runs/8247790348/job/22556708213?pr=919)!

**Pull request merged**